### PR TITLE
Crud (create, update, delete) application.

### PR DESCRIPTION
**Pull request by aakbarkhan**
## Description:

- To-Do List should be saved in local storage.
-  Deleting a task.
- "Clear all completed" button.
-  editing task descriptions.
-  adding a new task.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -73,12 +78,85 @@ export function display() {
...
+    e.preventDefault();
+    editbtn[i].innerText = 'save';
+    edittodo(i - 1);
+  });
+});
+function indexLoop() {
+  if (todoLists.length > 0) {
+    let i = 0;
+    while (i < todoLists.length) {
+      todoLists[i].index = i;
```
Comment (rloterh) [inline src/index.js]: - [x] New entries **do not match** the property index. The _property index_ must be set to the value of the _new array length_ (i.e. if a 5th task is added to the list, the index of that task should equal 5).

**Index mismatch in your application:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Aku, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- CRUD features are functioning as intended
- Functions effectively split into appropriate files
- Module import correctly implemented
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (aakbarkhan) [reply src/index.js]: Fixed, thank you.

Comment (henatan99) [reply src/index.js]: This issue has been addressed well. Nice job!

File: src/index.js [hunk 2]
```diff
@@ -73,12 +78,88 @@ export function display() {
...
+function edittodo(i) {
+  area[i].style.display = 'inline';
+  const line = document.querySelectorAll('.item');
+  line[i].style.display = 'none';
+  area[i].addEventListener('change', () => {
+    area[i].textContent = area[i].value;
+    todoLists[i].description = area[i].textContent;
+    addLocalStorage();
+  });
+}
```
Comment (henatan99) [inline src/index.js:157]: - [ ] OPTIONAL: When I edit a to-do item, It takes the text and hidden characters as textContent. So when I edit and hit 'enter', it takes the ↵ character along with the todo description text. Kindly fix this issue.  :pray: 

Screenshot from 2021-11-06 14-59-56

**Review by henatan99 — APPROVED** [review]
Hi @aakbarkhan >, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**